executor/src/main/scala/org/apache/amaterasu/executor/execution/actions/runners/spark/PySpark/PySparkRunner.scala

```diff
@@ -53,6 +54,7 @@ class PySparkRunner extends AmaterasuRunner with Logging {
 
     do {
       res = resQueue.getNext()
+      notifier.info(s"RES: $res")
```

Comment:
for notifier info (as opposed to debug messages), I would explain the log string a bit better. i.e "Result: " instead of "RES:"